lib/searchable_select.ex

```diff
@@ -197,10 +197,10 @@ defmodule SearchableSelect do
   end
 
   def prep_options(%{assigns: assigns} = socket, %{options: options}) do
-    gb_options =
-      Enum.reduce(options, :gb_trees.empty(), fn option, acc ->
-        normalised_label = assigns.label_callback.(option) |> normalise_string()
-        :gb_trees.insert(normalised_label, option, acc)
+    [_ , gb_options] =
+      Enum.reduce(options, [0, :gb_trees.empty()], fn option, [count, tree] ->
+        searchable_string = assigns.label_callback.(option) |> normalise_string()
+        [count + 1, :gb_trees.insert(count, {option, searchable_string}, tree)]
```

Comment:
to do least about of damage to test:

`        [count + 1, :gb_trees.insert("#{count}-#{searchable_string}", option, tree)]`